packages/fxa-settings/src/components/InputPhoneNumber/index.test.tsx

```diff
@@ -20,46 +19,39 @@ describe('InputPhoneNumber', () => {
     const options = screen.getAllByRole('option');
     expect(options.length).toBe(2);
 
-    // The first option should be United States, and it should be selected by default
-    expect(options[0]).toHaveTextContent('United States (+1)');
-    expect((options[0] as HTMLOptionElement).selected).toBe(true);
+    // expect list to be sorted alphabetically
+    expect(options[0]).toHaveTextContent('Canada (+1)');
+    expect(options[1]).toHaveTextContent('United States (+1)');
```

Comment:
Just wondering, did UX give this the thumbs up too? It was specifically requested at the time that US be the default. I'm cool with this and think it makes sense but just checking.

Reply:
Yes! Got sign off from Eduardo that divider is not needed if one of the options is pre-selected. (comments in ticket)